scripts/utils/migrate_includes.py

```diff
@@ -37,7 +37,7 @@ def update_includes(project, dry_run):
         content = ""
         migrate = False
         for line in lines:
-            m = re.match(r"^(.*)#include <(.*\.h)>(.*)$", line)
+            m = re.match(r"^(.*)#include [<\"](.*\.h)[>\"](.*)$", line)
```

Comment:
I don't think we should be processing `""` (local) includes. I think it's ok to edit the script to detect and fix them, though.

Reply:
Cool with that, I was on the edge for this, I'll drop that patch to the script and keep the other two.